src/contract-details/ContractReceipt.js

```diff
@@ -15,23 +15,30 @@ export default class ContractReceipt extends PureComponent {
 
 	render() {
 		const { contract, showLongcode, onTradeAgainClicked } = this.props;
+		const backendFailure = contract.validation_error && Object.keys(contract).length < 3;
```

Comment:
Is this a hack or a documented thing?
What significance has the 'contract keys less than 3'